mypy/nodes.py

```diff
@@ -587,6 +595,11 @@ class Var(SymbolNode, Statement):
     # parse for some reason (eg a silenced module)
     is_suppressed_import = False
 
+    FLAGS = [
+        'is_self', 'is_ready', 'is_initialized_in_class', 'is_staticmethod',
+        'is_classmethod', 'is_property', 'is_settable_property', 'is_suppressed_import'
```

Comment:
Old habits die hard, I guess :(

I think I typically remember when modifying an existing list of item, since the trailing comma is already there, but it hasn't quite sunken in for new lists yet.